packages/better-auth/src/plugins/oidc-provider/types.ts

```diff
@@ -284,6 +289,11 @@ export interface Client {
 	 * Whether the client is disabled or not.
 	 */
 	disabled: boolean;
+	/**
+	 * Whether to skip the consent screen for this client.
+	 * Only applies to trusted clients.
+	 */
+	skipConsent?: boolean;
```

Comment:
```suggestion
	skipConsentForTrustedClients?: boolean;
```

Suggestion: use a more explicit name to make it both a less concerning option, and more obvious.

Reply:
Though it's important to teach proper consent :)